proposals/stage-3/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -0,0 +1,320 @@
+# Cross Project Council (CPC) Charter
+
+## Section 1. Guiding Principle.
+
+The OpenJS Foundation will operate transparently, openly,
+collaboratively, and ethically. Project proposals, timelines, and status
+must not merely be open, but also easily visible to outsiders.
```

Comment:
Interpreted in a straightforward way, this sentence is meaningless. The phrase "easily visible to outsiders" is supposed to be in contrast to "open". Without further explanation, it's not clear what the distinction is supposed to be.

Reply:
Easy possible solutions would be to remove the sentence or to simply state: "Project proposals, timelines, and statuses must be open." Or: "Project proposals, timelines, and statuses must be easily visible." Or "easily visible to outsiders." Or "easily visible to anyone." Lots of options.

Reply:
I think we should expand on this in a follow up PR

Reply:
I believe this text came from an earlier doc (not sure if TSC governance, something from the JSF or possibly Kubernetes) so I agree we should expand in a follow on PR.

Reply:
Not going to include this in these changes. Lets do a follow up